extension/css/first-run.css

```diff
@@ -71,25 +71,25 @@ p.first-run-p {
 }
 
 a {
-  color: var(--blue4);
+  color: var(--relayBlue4);
   border-radius: 2px;
 }
 
 button:hover {
-  background-color: var(--blueHover);
+  background-color: var(--relayBlueHover);
 }
 
 button:active {
-  background-color: var(--blueActive);
+  background-color: var(--relayBlueActive);
 }
 
 a:hover {
-  color: var(--blueHover);
+  color: var(--relayBlueHover);
 }
 
 button:focus,
 a:focus {
-  box-shadow: var(--blueButtonFocus);
+  box-shadow: var(--relayButtonFocus);
 }
 
 a:active {
```

Comment:
Q: Just below this we have `color: var(--blueButtonActive);`, but I'm not sure if that should also have a `relay` prefix.